tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.hh

```diff
@@ -152,7 +152,7 @@ protected:
 
     // This function creates a string based on a vector of string inputs that is delimited by DataStringGlobals::NL by default, but any
     // delimiter can be passed in to this function. This allows for cross platform output string comparisons.
-    std::string delimited_string(std::vector<std::string> const &strings, std::string const &delimiter = "\n");
+    static std::string delimited_string(std::vector<std::string> const &strings, std::string const &delimiter = "\n");
```

Comment:
I kinda needed it be static, and it can be, so...

Reply:
Hmm.  OK.  I would have thought you could work around it without this change, but also it's fine.